src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const elementList = [...document.querySelectorAll('.population')];
+
+const elementNumbers = elementList.map((el) => {
+  const populationText = el.textContent.trim();
+
+  const populationNumber = parseInt(populationText.replace(/,/g, ''));
```

Comment:
Ensure that the text content of the elements is always a valid number. If the text content is not a valid number, `parseInt` will return `NaN`, which could affect the total and average calculations. Consider adding a check to handle such cases.